README.md

```diff
@@ -310,6 +325,54 @@ function assignId (req, res, next) {
 }
 ```
 
+#### arguments to custom token formats
+
+Example of a custom token format that uses an argument. It will output the
```

Comment:
This is a solid point. I've had similar experiences on other projects. How about a more absurd example such as _"is it christmas?"_

> Example of a custom token format that uses an argument. It will output if the time
> of the current request is christmas.

```js
var express = require('express')
var morgan = require('morgan')

morgan.token('is-holiday', function getId (req, res, arg) {
  var date = new Date()
  var shortDate = `${date.getMonth() + 1}/${date.getDate()}`;
  var holiday = arg.toLowerCase().replace(/'/g, '');

  var holidays = {
    '1/1': 'new years day',
    '2/14': 'valentines day',
    '5/5': 'cinco de mayo',
    '10/31': 'halloween',
    '12/25': 'christmas',
    '12/31': 'new years eve'
  };

  var isHoliday = 'Nope';
  if (holidays[shortDate] === holiday) {
    isHoliday = 'Yup';
  }

  return String(isHoliday);
})

var app = express()

app.use(morgan('Is it Christmas? :is-holiday[christmas] :response-time'))

//
// Alternatively we could ask if it is Halloween
//
app.use(morgan('Is it Halloween? :is-holiday[halloween] :response-time'))


app.get('/', function (req, res) {
  res.send('hello, world!');
})
```